app/components/access_panels/location_item_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= tag.tr class: classes, data: { barcode: item.barcode } do %>
+  <td class="callnumber">
+    <%= item.callnumber %>
+  </td>
+  <% if render_item_note? %>
+    <td class="public-note">
+      <% if item.public_note.present? %>
+        <div class='public-note note-highlight'>Note: <%= inject_line_break_opportunities item.public_note %></div>
+      <% end %>
+    </td>
+  <% end %>
+  <% if render_item_details? %>
+    <td class="item-availability" data-live-lookup-id="<%= document.live_lookup_id %>" data-status-target=".availability-icon" data-item-id="<%= item.live_lookup_item_id if item.live_status? %>" <%= "data-request-url='#{helpers.request_url(document, library: item.library, location: item.home_location, barcode: item.barcode)}'".html_safe if render_real_time_availability_request_link? %>>
+      <span class="availability-icon-wrapper">
+        <i class="availability-icon <%= item.status.availability_class %>"></i>
+        <span data-available-text="<%= t('searchworks.availability.available') %>" data-unavailable-text="<%= t('searchworks.availability.unavailable') %>" class='status-text'>
+          <%= availability_text %>
+        </span>
+      </span>
+      <span class="current-location">
+        <%= current_location_text %>
+      </span>
+      <% if item.on_reserve? %>
+        <%= item.loan_period %>
+      <% end %>
+      <% if render_item_level_request_link? %>
+        <span class="request-link">
+          <%= render item.request_link %>
+        </span>
+      <% end %>
+    </td>
+  <% else %>
+    <td></td>
```

Comment:
I hope this is the only markup difference between index/show pages; we currently show availability for bound-withs on the index page but not the show page (weird), but Sarah's ok with consistency.

Reply:
Agree that that's weird (if anything seems like it'd be the reverse) but +1 for consistency